eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -38,4 +38,9 @@
     <UsagePattern IdentityGlob="System.Reflection.MetadataLoadContext/*7.0.0*" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/*7.0.0*" />
   </IgnorePatterns>
+  <Usages>
```

Comment:
These will break source-build as customers require building offline without prebuilts.  @jaredpar started a thread with the source-build team on this including @mmitche and @mthalman.

Reply:
Yes, I'm trying a different approach now, thanks.